### PR TITLE
Effectively disable CI on this feedstock

### PR DESCRIPTION
**Pull request by xhochy**
The CI jobs for the full matrix always timeout after six hours. As the matrix is quite large, disable it so that it doesn't block other jobs from running.

### CODE REVIEW

File: conda-forge.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+azure:
+  settings_linux:
+    timeoutInMinutes: 1
+  settings_osx:
+    timeoutInMinutes: 1
```
Comment (hmaarrfk) [inline conda-forge.yml:5]: Amazing!

shall we revert the pinning changes?

Comment (xhochy) [reply conda-forge.yml:5]: I would like to do this in a separate PR. They make take some time due to `nvcc` weirdness and I simply want to minimise the load impact our work on this feedstock has.

Comment (hmaarrfk) [reply conda-forge.yml:5]: i meant

```
git checkout main -- .ci_support
```
to undo the changes in the ci_support files that relate to pinning upgrades that didn't happen.

Comment (xhochy) [reply conda-forge.yml:5]: Ah, got it!

**Pull request merged**